packages/nimble-components/src/rich-text-editor/models/italics-button.ts

```diff
@@ -0,0 +1,31 @@
+import { html } from '@microsoft/fast-element';
+import type { Editor } from '@tiptap/core';
+import { EditorButton } from './editor-button';
+import { iconItalicITag } from '../../icons/italic-i';
+
+/**
+ * Derived class from EditorButton for handling italics button events.
+ */
+export class ItalicsButton extends EditorButton {
+    public constructor(public override tiptapEditor: Editor) {
+        super(tiptapEditor);
+        this.class = 'italics';
+        this.iconLabel = 'italics';
```

Comment:
Label string needs to have first letter capitalized in all button model classes? 

```suggestion
        this.iconLabel = 'Italics';
```

Reply:
I have updated the name in all button classes